README.md

```diff
@@ -1,82 +1,65 @@
 
 [![CI](https://github.com/Quantco/spox/actions/workflows/ci.yml/badge.svg)](https://github.com/Quantco/spox/actions/workflows/ci.yml)
-[![Documentation](https://img.shields.io/badge/docs-latest-success?style=plastic)](https://docs.dev.quantco.cloud/qc-github-artifacts/Quantco/spox/latest/index.html)
+[![Documentation Status](https://readthedocs.org/projects/spox/badge/?version=latest)](https://spox.readthedocs.io/en/latest/?badge=latest)
 
-Spox is a Python framework for constructing [ONNX](https://github.com/onnx/onnx/) computational graphs.
+Spox makes it easy to construct [ONNX](https://github.com/onnx/onnx/) models through clean and idiomatic Python code.
 
-Spox:
+## Why use Spox?
 
-- Implements the ONNX standard while also allowing Pythonic code.
-  - Spox follows conventions set out by numpy and other Python libraries.
-- Enforces the strong type system of ONNX, by raising errors with Python tracebacks to the offending operator.
-  - Checks are performed as eagerly as possible!
-- Supports the entirety of modern opsets, including features like subgraphs (control flow) and types other than tensors (like sequences and optionals).
-  - Standard operators all have typed Python signatures and docstrings!
-- Is designed for predictability. No mutable types are passed around, so it's difficult to invalidate the graph accidentally.
-  - If it's legal Spox, it (should be) legal ONNX!
-
-The main goal of Spox is to provide a robust and Pythonic framework for developing libraries building ONNX graphs, such as converters or other custom applications.
+Converting a trained model into ONNX entails replicating its inference logic with ONNX operators.
+In the past that constituted a major undertaking.
+Based on those experiences we designed Spox from the ground up to make the process of writing converters (and ONNX models in general) as easy as possible.
 
 ## Installation
 
-Spox is published on conda-forge and can be installed as expected:
+With pip:
 
 ```bash
-conda install spox
+pip install spox
 ```
 
-## Getting started
-
-### Constructing
-
-In Spox, most of the time you'll be working with `Var` objects - **variables**. You either create them yourself as arguments for a model, or receive them from another source, for example when you're writing a converter in a library.
-
-You may print out the `Var` or check its `Var.type` to learn a bit more about it.
-
-To perform operations on a `Var`, use an _opset_ (operator set) module like `spox.opset.ai.onnx.v17` or `spox.opset.ai.onnx.ml.v3` - which correspond to `ai.onnx@17` and `ai.onnx.ml@3`, the standard opsets. These are pre-generated for you and you may import them as required.
-
-For instance, using the default opset you could write a function that given two variables returns their [geometric mean](https://en.wikipedia.org/wiki/Geometric_mean):
-
-```python
-from spox import Var
-from spox.opset.ai.onnx import v17 as op
+Spox is also available on conda-forge and can be installed as expected:
```

Comment:
```suggestion
There is also a package available on conda-forge:
```
Also, technically this isn't quite there yet.

Reply:
Yes, coming up.